packages/devtools_app/lib/src/screens/network/offline_network_data.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_shared/devtools_shared.dart';
+
+import '../../shared/http/http_request_data.dart';
+import '../../shared/primitives/utils.dart';
+import 'network_model.dart';
+
+/// Class to encapsulate offline data for the [NetworkController].
```

Comment:
nit: dart doc style, add blank line between the first sentence of dart doc and the remaining content

```
/// First sentence.
///
/// Remaining content.
```

Reply:
done